gallery/src/components/NxTextLink/NxTextLinkInternalExample.tsx

```diff
@@ -11,7 +11,7 @@ import { NxCode, NxP, NxTextLink } from '@sonatype/react-shared-components';
 export default () =>
   <>
     <NxP>
-      <NxTextLink href="#/pages/nx-tile">nx-tile documentation page</NxTextLink>
+      <NxTextLink href="#/pages/Tile">Tile documentation page</NxTextLink>
```

Comment:
Not sure if there is a workout around here, but this brings you to the middle of the page instead of the top

Reply:
I found this issue as well and created a ticket for it below:
https://issues.sonatype.org/browse/RSC-1089

Since this issue should not be fixed during this ticket. I'm closing this conversation.